repo/content/committed_read_manager.go

```diff
@@ -620,12 +624,5 @@ func NewSharedManager(ctx context.Context, st blob.Storage, prov format.Provider
 		return nil, errors.Wrap(err, "error setting up read manager caches")
 	}
 
-	sm.indexesLock.Lock()
-	defer sm.indexesLock.Unlock()
-
-	if err := sm.loadPackIndexesLocked(ctx); err != nil {
```

Comment:
what was the original intention of this ? also, why  is this not needed anymore ?

Reply:
ah

```
This is achieved by removing the index loading during kopia repo status and postponing index loading through lazy initialization.
```